deploy/helm/grafana-operator/templates/deployment.yaml

```
@@ -85,6 +85,7 @@ spec:
             {{- if .Values.leaderElect }}
             - --leader-elect
             {{- end }}
+            - --max-concurrent-reconciles=1
```

Looks like the max-concurrent-reconciles is hardcoded instead of using values.
```suggestion
            - --max-concurrent-reconciles={{ .Values.maxConcurrentReconciles }}
```